spec/lib/administrate/resource_resolver_spec.rb

```diff
@@ -7,23 +7,23 @@
   describe "#dashboard_class" do
     it "handles global-namepsace models" do
```

Comment:
`"handles namepsace models"`?
I am not sure. User model in a top/global namespace, I am miss something?